tests/baselines/reference/commentLeadingCloseBrace.js

```diff
@@ -0,0 +1,24 @@
+//// [commentLeadingCloseBrace.ts]
+declare function commentedParameters(...args): any;
+
+function ifelse() {
+    if (commentedParameters(1, 2)) {
+        /*comment1*/
+        commentedParameters(3, 4);
+        /*comment2*/
+    } else {
+        commentedParameters(5, 6);
+    }
+}
+
+//// [commentLeadingCloseBrace.js]
+function ifelse() {
+    if (commentedParameters(1, 2)) {
+        /*comment1*/
+        commentedParameters(3, 4);
+    /*comment2*/

```

Comment:
We might want to investigate whether it makes sense to emit leading/trailing comments of a position in the `emitNodeList` function in `emitter.ts`.